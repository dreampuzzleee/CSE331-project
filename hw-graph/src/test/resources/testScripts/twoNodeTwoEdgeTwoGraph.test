# Tests graph creation/viewing with two graph ('A' and 'B') with two nodes and two edges.

# Create a graph
CreateGraph A
CreateGraph B

# Add a pair of nodes
AddNode A n1
AddNode B n2

# Add an edge
AddEdge A n1 n1 e1
AddEdge B n2 n2 e2

# Print all nodes in the graph
ListNodes A
ListNodes B

# Print all child nodes of n1, n2 with outgoing edge
ListChildren A n1
ListChildren B n2