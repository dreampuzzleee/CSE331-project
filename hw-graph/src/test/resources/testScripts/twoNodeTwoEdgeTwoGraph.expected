# Tests graph creation/viewing with two graph ('A' and 'B') with two nodes and two edges.

# Create a graph
created graph A
created graph B

# Add a pair of nodes
added node n1 to A
added node n2 to B

# Add an edge
added edge e1 from n1 to n1 in A
added edge e2 from n2 to n2 in B

# Print all nodes in the graph
A contains: n1
B contains: n2

# Print all child nodes of n1, n2 with outgoing edge
the children of n1 in A are: n1(e1)
the children of n2 in B are: n2(e2)